source/utils.ts

```diff
@@ -35,3 +35,7 @@ export function getTypeCheckerAndNodeMap(context: Rule.RuleContext) {
     typeChecker
   };
 }
+
+export function getParent<T extends es.Node = es.Node>(node: es.Node): T {
```

Comment:
Same opinion re: using explicit type parameters for assertions as mentioned [here](https://github.com/cartant/eslint-plugin-rxjs/pull/12/files#r312771251).